src/server/json_family_test.cc

```diff
@@ -952,8 +952,10 @@ TEST_F(JsonFamilyTest, MGet) {
   resp = Run({"JSON.SET", "json2", ".", json[1]});
   ASSERT_THAT(resp, "OK");
 
+#ifndef SANITIZERS
```

Comment:
is it a good solution? Do you know why it failed?

Reply:
I am fairly certain that it's a false positive. I thought it was ADL and a funky conversion somewhere but I was wrong on that. I will spent a little bit time on this and ping back just to be on the safe side

Reply:
It is a false positive, I stepped it through in debug build and everything is fine in the flow. I switched off the check